Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -1368,6 +1368,20 @@ static AdditionalFonts additionalFonts(const Vector<URL>& fontURLs, std::optiona
     return additionalFonts;
 }
 
+static void addUserInstalledFontURLs(NSString *path, Vector<URL>& fontURLs)
+{
+    RetainPtr enumerator = [NSFileManager.defaultManager enumeratorAtPath:path];
+
+    for (NSString *font in enumerator.get()) {
+        if ([font hasSuffix:@"ttf"] || [font hasSuffix:@"ttc"] || [font hasSuffix:@"otf"]) {
```

Comment:
This seems very surprising, because it's just a suffix check and isn't case-insensitive.

I would expect use of pathExtension, strict equality instead of substring equality, and case-insensitive comparison...

Reply:
Ah, good catch, Tim! I will address in a follow-up.

Thanks for reviewing!